source/includes/_endpoint-dataset.md

```diff
@@ -686,96 +686,66 @@ Refer to the options described on the table above for the `csv` format to change
 
 
 ##### Match
-`GET /datasets/match{?datasets={id}&datasets={id}[&datasets={id}]+`
+```http
+POST /datasets/match HTTP/1.1
+Host: app.crunch.io
+Content-Type: application/json
+
+{  
+   "graph":[  
+      {  
+         "datasets":[  
+            "http://local.crunch.io:49847/api/datasets/06a47aff9b2343ea9ad5e92a05b94fef/",
+            "http://local.crunch.io:49847/api/datasets/9f4671e0be8746e889e39be1d0fd0b8d/",
+            "http://local.crunch.io:49847/api/datasets/d1378d19c2824360b6d0d898b55ddf8e/"
+         ]
+      }
+   ],
+   "element":"shoji:order"
+}
+```
 
 ```json
 
 {
-  "self": "http://local.crunch.io/api/datasets/match/?datasets=http://app.crunch.io/api/datasets/26df3c304/&datasets=http://app.crunch.io/api/datasets/3e03136be/", 
-  "value": {
     "graph": [
-      {
-        "datasets": [
-          {
-            "archived": false, 
-            "description": "", 
-            "end_date": null, 
-            "labels": null, 
-            "creation_time": "2017-06-29T01:49:30.831000", 
-            "start_date": null, 
-            "modification_time": "2017-06-29T01:49:30.774000", 
-            "self": "http://app.crunch.io/api/datasets/26df3c304/", 
-            "projects": [], 
-            "name": "test_datasets_match_1"
-          }, 
-          {
-            "archived": false, 
-            "description": "", 
-            "end_date": null, 
-            "labels": null, 
-            "creation_time": "2017-06-29T01:49:32.261000", 
-            "start_date": null, 
-            "modification_time": "2017-06-29T01:49:32.207000", 
-            "id": "28b773839ac747bb834b8b4713f77c06", 
-            "projects": [], 
-            "name": "test_datasets_match_2"
-          }
-        ]
-      }, 
-      {
-        "variables": [
-          {
-            "description": "Numeric variable with value labels", 
-            "category_names": [
-              "red", 
-              "green", 
-              "blue", 
-              "4", 
-              "8", 
-              "9", 
-              "No Data"
+        {
+            "matches": [
+                [
```

Comment:
Please also add this change to that ticket:
https://github.com/Crunch-io/backend-http/pull/728/files#diff-ebceaf08643a7a452d686bf57136b7e8R121 should be shoji:entities